src/IssueCreator/IssueManager.cs

```diff
@@ -15,13 +18,15 @@ namespace IssueCreator
 {
     public class IssueManager
     {
-        ZenHubClient _zenHubClient;
-        GitHubClient _githubClient;
-        MemoryCache _cache = new MemoryCache(new MemoryCacheOptions());
+        private ZenHubClient _zenHubClient;
+        private GitHubClient _githubClient;
+        private MemoryCache _cache = new MemoryCache(new MemoryCacheOptions());
+        private Dictionary<string, Type> _cacheEntries = new Dictionary<string, Type>(); //keeps track of the keys added to the cache
```

Comment:
This should be removed -- it was initially used to keep track of which new keys are cached, but in the end the data is cached as it comes in, removing the need for this.